ci/aws/aws-cluster.lokocfg.envsubst

```diff
@@ -8,12 +8,13 @@ variable "os_channel" {
 }
 
 cluster "aws" {
-  asset_dir    = pathexpand(var.asset_dir)
-  cluster_name = "$CLUSTER_ID"
-  dns_zone     = "$AWS_DNS_ZONE"
-  dns_zone_id  = "$AWS_DNS_ZONE_ID"
-  os_channel   = var.os_channel
-  ssh_pubkeys  = ["$PUB_KEY"]
+  asset_dir        = pathexpand(var.asset_dir)
+  cluster_name     = "$CLUSTER_ID"
+  controller_count = 3
```

Comment:
We could also use just 2.

Reply:
right, that should be enough, I'll change.